pom.xml

```diff
@@ -16,7 +16,7 @@
         <maven.compiler.target>11</maven.compiler.target>
 
         <spring-boot-dependencies.version>2.5.4</spring-boot-dependencies.version>
-        <c8y.version>1010.0.0</c8y.version>
```

Comment:
please update to 10.15.334.0  to avoid merge conflicts.

Reply:
done